CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ This can also be enabled programmatically with `warnings.simplefilter('default',
 * support gradients as fill or stroke colors in the drawing API - _cf._ [PR #1539](https://github.com/py-pdf/fpdf2/pull/1539): [link to documentation](https://py-pdf.github.io/fpdf2/Drawing.html#compositing-operations)
 * support for gradient spread methods (PAD, reflect and repeat)
 * support for sweep (mesh) gradients
+* support for SVG `<linearGradient>` and `<radialGradient>` elements - _cf._ [issue #1580](https://github.com/py-pdf/fpdf2/issues/1580) - thanks to @Ani07-05
```

Comment:
Please rebase your PR or merge the latest changes on your branch. Your changelog entry should now be under version 2.8.6